README.md

```diff
@@ -34,11 +44,18 @@ You can sign up for your own app id at [https://dashboard.branch.io](https://das
 
 #### Unity Scene and Branch Parameters
 
-To allow Branch to configure itself, you must add a BranchPrefab asset to your first scene. Simply drag into your first scene, and then specify your `APP_KEY`, `PATH_PREFIX` and `APP_URI` in the properties.
+To allow Branch to configure itself, you must add a BranchPrefab asset to your first scene. Simply drag into your first scene, and then specify your `APP_KEY`, `APP_PREFIX` and `PATH_PREFIX`, `CUSTOM_DOMAIN`, `APP_LINKS` in the properties.
 
-* `branchKey`: This is your Branch key from the dashboard
-* `androidPathPrefix`: This is your Branch android path prefux [Read more](https://github.com/BranchMetrics/Android-Deferred-Deep-Linking-SDK/blob/master/README.md#leverage-android-app-links-for-deep-linking)
-* `branchUri`: This is the URI scheme you would like to use to open the app. This must be the same value as you entered in [the Branch link settings](https://dashboard.branch.io/#/settings/link) as well. Please do *not* include the `://` characters.
+* `Simulate Fresh Installs`: This is flag that enable or desable debud mode (in debug mode e your app will simulate fresh install each time, just for testing)
+* `Test Mode` : Switch set of parameters, if "Test mode" is enabled then app will use "test" set of parameters, else app will use "live" set of parameters
+* `APP_KEY`: This is your Branch key from the dashboard
+* `APP_URI`: <b>Keeped for supporting old realisation of link scheme.</b> This is the URI scheme you would like to use to open the app. This must be the same value as you entered in [the Branch link settings](https://dashboard.branch.io/#/settings/link) as well. Please do *not* include the `://` characters.
+* `PATH_PREFIX`: <b>Keeped for supporting old realisation of link scheme.</b> This is your Branch android path prefux [Read more](https://github.com/BranchMetrics/Android-Deferred-Deep-Linking-SDK/blob/master/README.md#leverage-android-app-links-for-deep-linking)
+* `CUSTOM_DOMAIN` : <b>Keeped for supporting old realisation of link scheme.</b> This is your custom domain.
+* `APP_LINKS` : This is your app links.
```

Comment:
Still not totally clear. Sorry for being dense. When we refer to a 'Custom Domain', we're usually referring to what our friend Acimaz added in the App Links field: ayhan.meistercody.com. Normal domains are either xxxx.app.link and xxxx-alternate.app.link OR bnc.lt for old apps.

My question for you as I write up the documentation, is it ok for me to just tell people to put all 3 of these domains in the App Links field. Would all 3 get put into the manifest on Android and into the entitlements file on iOS if so?